RELEASE-NOTES.md

```diff
@@ -152,6 +152,30 @@ Including CS path: add following sentence to `advanced.config`:
 ]}.
 ```
 
+### Notes on upgrading Riak to 2.0
+
+Riak has updated its underlying Bitcask storage data format at 2.0. At
+the first start of Riak after upgrade, it involves implicit data
+format upgrade conversion, which means reading all data and writing
+down to another file. This might lead to disk load - the duration of
+upgrade will depend on the amount of data stored in bitcask and IO
+performance of underlying disk.
```

Comment:
This paragraph explains only downside. I want one addition sentence for why format change was needed.